.gitignore

```diff
@@ -7,6 +7,7 @@ node_modules/
 # Environment
 .env*
 !.env.example
+!.env.integration
```

Comment:
I think that now makes more sense just to ignore the `.env`:

```
# Environment
.env
```

I've just pushed a commit with this change :)